Modules/socketmodule.c

```diff
@@ -1635,7 +1635,7 @@ getsockaddrarg(PySocketSockObject *s, PyObject *args,
                 Py_TYPE(args)->tp_name);
             return 0;
         }
-        if (!PyArg_ParseTuple(args, "O&i|II",
+        if (!PyArg_ParseTuple(args, "O&i|II:getsockaddrarg",
```

Comment:
ISTM that currently the error messages are inconsistent:
```
>>> socket.socket(family=socket.AF_INET6).bind(('::1', -1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: getsockaddrarg: port must be 0-65535.
>>> socket.socket(family=socket.AF_INET6).bind(('::1', -1 << 1000))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: Python int too large to convert to C long
```

but I can leave the format as `"O&i|II"`, and just add:
```
        assert(PyErr_Occurred());
        if (PyErr_ExceptionMatches(PyExc_OverflowError)) {
            PyErr_SetString(PyExc_OverflowError,
                            "getsockaddrarg: port must be 0-65535.");
```
or do you think we should leave this change for bpo-29832?